tests/unit/test_wheel.py

```diff
@@ -243,9 +245,8 @@ def test_unpack_wheel_no_flatten(self):
         from tempfile import mkdtemp
         from shutil import rmtree
 
-        filepath = '../data/packages/meta-1.0-py2.py3-none-any.whl'
-        if not os.path.exists(filepath):
-            pytest.skip("%s does not exist" % filepath)
+        filepath = os.path.join(DATA_DIR, 'packages',
```

Comment:
You can use the data fixture here -- then this line would become `data.packages / 'meta-1.0-py2.py3-none-any.whl'`.